### ISSUE TITLE
No new citizens on new colony without recruiting.  

### ISSUE THREAD

**Issue by BrewingCoder** [ldtteam/minecolonies#4766]
### Minecolonies version 

Version: 0.11.881-ALPHA  Forge 1.15.2-31.1.47

### Expected behavior

 children being born, new Citizens
- 

### Actual behaviour

20 minecraft days.  colony at 5 of 9 citizens.  No new babies or citizens.   

- 

Link to exceptions being thrown in log

### Steps to reproduce the problem

Nothing really, just have a town with available space and sit around and wait :)

1. 
2. 
3. 
4. ... Profit

Config:
#Average citizen respawn interval (in seconds). [Default: 60]
	#Range: 10 ~ 600
	citizenrespawninterval = 60

### Logs: Latest.log and crashlog at https://gist.github.com/

- latest.log: 
FYI both latest and full debug are in this Gist, latest is at bottom.

https://gist.github.com/BrewingCoder/0ecc24af1369fd4ee7d24257bf5cb371

- crashlog:
N/A no crash.

**Comment by someaddons** [comment]
I think you maybe send the wrong log, don't see any exceptions in that one

**Comment by Raycoms** [comment]
I believe you have no woman and man in the same building to procreate?

**Comment by BrewingCoder** [comment]
everyone was in the same building, I am pretty sure I only had one citizen hut. 

**This can be closed**.  I think the world was corrupted.  I created it on a single player instance (to get the terraforged world gen the way I wanted it, nice and flat) then moved it over to the dedicated server.   There were quite a few mods that the server instance did not have.

I just created a fresh world using the server to generate it and creatively placed an L5 citizen hut, and it is working properly.
